docs/basic/index.rst

```diff
@@ -97,23 +97,15 @@ the CrateDB package repository::
     # Import the public GPG key for verifying the package signatures.
     curl -sS https://cdn.crate.io/downloads/deb/DEB-GPG-KEY-crate | sudo tee /etc/apt/trusted.gpg.d/cratedb.asc
 
-    # Compute CrateDB package repository location.
-    [[ $(lsb_release --id --short) = "Debian" ]] && repository="apt"
-    [[ $(lsb_release --id --short) = "Ubuntu" ]] && repository="deb"
-    distribution=$(lsb_release --codename --short)
-
-    # Register with the CrateDB package repository.
-    echo "deb [signed-by=/etc/apt/trusted.gpg.d/cratedb.asc arch=amd64] https://cdn.crate.io/downloads/${repository}/stable/ ${distribution} main" \
-        | sudo tee /etc/apt/sources.list.d/cratedb.list
+   # Add CrateDB repository to Apt
+   sh$ echo "deb https://cdn.crate.io/downloads/debian/stable/ default main" |
+      sudo tee /etc/apt/sources.list.d/crate-stable.list
 
 .. NOTE::
 
-    CrateDB provides both *stable release* and *testing release* channels. To
-    use the testing channel, replace ``stable`` with ``testing`` in the command
-    line above. You can read more about the `release workflow`_.
-
-    The walkthrough is based on the ``sudo`` program. If it is not installed on
```

Comment:
I removed this sudo note because it's incomplete. If you're not familiar with `sudo`, then installing it is likely not enough because you also need to ensure the user then has permission to use it for package installation.

I think we can assume some basic system administration knowledge, and if not we should find some good resource we can link to.